### PR TITLE
Add a local course on getting started with NFTs.

### PR DESCRIPTION
**Pull request by jordanmack**
This PR adds a new course on how to get started with NFTs in the Nervos ecosystem, which is aimed at dapp developers who are not familiar with Nervos.

### CODE REVIEW

**Comment by jordanmack** [comment]
@Flouse I worked on a new course for CKB Academy. It works locally, but something is wrong with the preview deployment. On the local courses, none of the "Next chapter" buttons are working. Do you have any ideas on what might cause this?

File: site/src/content/courses/nft-getting-started/index.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { Course, CourseSource, CourseType } from '~/types/course';
+import { contributorsData } from '~/data/contributors.data';
+
+const nftQuickStart: Course = {
+  id: 'nft-getting-started',
+  author: [
+    // see https://github.com/RetricSu/zero2ckb-web/graphs/contributors
```
Comment (Flouse) [inline site/src/content/courses/nft-getting-started/index.ts]:
```suggestion
```

File: site/src/content/courses/nft-getting-started/chapter_2.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Standards Introduction
+
+## The History of NFT Standards 
+
+The first early versions of NFTs were first introduced in 2014, but it took several years more for NFT popularity to take off. The first wildly successful NFT dapp was CryptoKitties, which launched in late 2017. This NFT game was so popular that it heavily congested the Ethereum network, causing it to reach an all-time high in the number of transactions and slowed down the network to a point where it temporarily became unusable unless you paid a high transaction fee.
+
+The first official NFT standard to be finalized was ERC721 in early 2018, which was based on what was used for CryptoKitties a few months prior. A few months after that, ERC1155 would be released, which added the ability to create multiple NFT token types in a single contract which streamlined the process for developers of creating multiple NFTs. Led by the success of CryptoKitties, ERC721 and ERC1155 paved the way for future developers to innovate using NFTs without having to "reinvent the wheel".
```
Comment (Flouse) [inline site/src/content/courses/nft-getting-started/chapter_2.mdx]:
```suggestion
The first official NFT standard to be finalized was ERC721 in early 2018, which was based on what was used for CryptoKitties a few months prior. A few months after that, ERC1155 was released, which added the ability to create multiple NFT token types in a single contract which streamlined the process for developers of creating multiple NFTs. Led by the success of CryptoKitties, ERC721 and ERC1155 paved the way for future developers to innovate using NFTs without having to "reinvent the wheel".
```

File: site/src/content/courses/nft-getting-started/chapter_2.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## NFT Permanence
+
+Most NFTs contain a reference to the content they represent but do not contain the data itself on-chain. This is especially true of many of the NFTs that represent images or music. The on-chain component represents proof of ownership, but only a URL to the data is contained on-chain. The data itself is often contained on a centralized hosting service such as Amazon S3 or a semi-decentralized protocol like IPFS. This creates a scenario where the NFT may outlive the content that the NFT represents if for some reason the data on the centralized hosting service becomes unavailable.
```
Comment (Flouse) [inline site/src/content/courses/nft-getting-started/chapter_2.mdx]:
```suggestion
Most NFTs contain a reference to the content they represent but do not contain the data itself on-chain. This is especially true for many of the NFTs that represent images or music. The on-chain component represents proof of ownership, but only a URL to the data is contained on-chain. The data itself is often contained on a centralized hosting service such as Amazon S3 or a semi-decentralized protocol like IPFS. This creates a scenario where the NFT may outlive the content that the NFT represents if for some reason the data on the centralized hosting service becomes unavailable.
```

**Comment by Flouse** [comment]
> It works locally, but something is wrong with the preview deployment.

image



This may only be a problem related to the Vercel preview version, which is blocked by a `vercel-feedback` layer.

If I manually hide the `div` in the  image above, the `Next chapter` button works fine.

It might be a bug of Vercel.live.
I think we could ignore it for now.

**Review by Flouse — APPROVED** [review]

**Review by oriole-pub — APPROVED** [review]

**Pull request merged**